plugins/GSdx/GSRendererHW.cpp

```diff
@@ -186,6 +186,42 @@ void GSRendererHW::SetGameCRC(uint32 crc, int options)
 	GSRenderer::SetGameCRC(crc, options);
 
 	m_hacks.SetGameCRC(m_game);
+
+	// Code for Automatic Mipmapping. Relies on game CRCs.
+	if (theApp.GetConfigI("mipmap_hw") == -1) {
+		switch (m_game.title) {
+			// Games requiring basic mipmapping
```

Comment:
Instead of having comments for explaining each of the values of ``mipmap_hw``, I'd prefer if an ``enum class`` is added for it similar to CRC hack level.